__tests__/DefaultApi.spec.ts

```diff
@@ -0,0 +1,412 @@
+        const request = {
+            _from: "_from_example",
+            to: "to_example",
+            text: "Example Text"
+        }
+        return apiInstance.sendAnSmsMessage(request).then((data) => expect(data).toBeInstanceOf(MessageResult))
+    });
+
+    //Error on this test due to not being able to find an assertion for httpFile being returned
+    test('#streamARecordingFile', async () => {
+        const recordingId = "recordingId_example"
+        return apiInstance.streamARecordingFile(recordingId).then((httpFile) => {
+            expect(httpFile.data).toBeInstanceOf(Buffer)
+            expect(typeof httpFile.name).toBe("string")
+        })
+    });
+
+    test('#updateAConference', async () => {
+        const conferenceId = "conferenceId_example"
+        const request: UpdateConferenceRequest = {
+              alias: "alias_example",
+              playBeep: PlayBeep.ALWAYS,
+              status: UpdateConferenceRequestStatus.EMPTY,
+        }
+        return apiInstance.updateAConference(conferenceId, request).then((data) => expect(data).toBeInstanceOf(ConferenceResult))
+    });
+
+    test('#updateALiveCall', async () => {
+        const callId = "callId_example"
+        const request: UpdateCallRequest = {
+              status: UpdateCallRequestStatus.CANCELED,
+        }
+        return apiInstance.updateALiveCall(callId, request)
+    });
+
+    test('#updateAParticipant', async () => {
+        const callId = "callId_example"
+        const conferenceId = "conferenceId_example"
+        return apiInstance.updateAParticipant(callId, conferenceId).then((data) => expect(data).toBeInstanceOf(ConferenceParticipantResult))
+    });
+
+    test('#updateAQueue', async () => {
+        const queueId = "queueId_example"
+        const queueRequest: QueueRequest  =  {
+            alias: "alias_example",
+            maxSize: 100,
+        }
+        return apiInstance.updateAQueue(queueId, queueRequest).then((data) => expect(data).toBeInstanceOf(QueueResult))
+    });
+
+    test('#updateAnAccount', async () => {
+        const accountRequest: AccountRequest  =  {
+            alias: "alias_example",
+            label: "label_example",
+        }
+        return apiInstance.updateAnAccount(accountRequest)
+    });
+
+    test('#updateAnApplication', async () => {
+        const accountRequest: AccountRequest  =  {
+            alias: "alias_example",
+            label: "label_example",
+        }
+        return apiInstance.updateAnAccount(accountRequest)
+    });
+
+    test('#updateAnIncomingNumber', async () => {
+        const phoneNumberId = "phoneNumberId_example";
+        const incomingNumberRequest: IncomingNumberRequest  =  {
+            applicationId: "applicationId_example",
+            alias: "alias_example",
+        }
+        return apiInstance.updateAnIncomingNumber(phoneNumberId, incomingNumberRequest).then((data) => expect(data).toBeInstanceOf(IncomingNumberResult))
+    });
```

Comment:
tests from line 83 no longer contain `expect.assertions(..)` is that intentional?

Reply:
this depends on whether there is a then statement, some of them do not have a then statement so we cannot expect an assertion, I'll re-add in this line for the ones expecting an assertion

Reply:
shouldn't all methods have a .then ? 
can we just use async/await instead of async + .then?

Reply:
not all methods, because some methods return voids, do we want to assert against voids?